### ISSUE TITLE
TV Category

### ISSUE THREAD

**Issue by NeGroRA** [rickeylohia/plugin.video.stalkervod#22]
Hello, why is it that no channel in the TV category is starting, but the movies start normally? 

I get an error message "playback failed"

**Comment by NeGroRA** [comment]
[https://zapodaj.net/images/f35fa0865308e.jpg](url)

**Comment by NeGroRA** [comment]
https://zapodaj.net/images/f35fa0865308e.jpg

**Comment by rickeylohia** [comment]
Hi @NeGroRA there are sometime differences in how each IPTV provider works. The plugin was tested on the couple of  providers I was able to get info on.
Can you enable debug logs in Kodi and provide the logs, can try and check why its not working for your provider.
Also if you can provide the details of your IPTV, I can try and test and fix the issue.
Let me know if you are ok to provide the details, will share my discord user id where you can share the details instead of sharing here on github

**Comment by rickeylohia** [comment]
@NeGroRA since there hasn't been any response, closing this.
feel free to re-open or create a new issue

**Comment by NeGroRA** [comment]
@rickeylohia

**Comment by NeGroRA** [comment]
Discord

**Comment by rickeylohia** [comment]
Added you on discord, my username is fierygreywind

**Comment by rickeylohia** [comment]
@NeGroRA this has been fixed with v1.2.0
will be available once https://github.com/xbmc/repo-plugins/pull/4699 is merged
